paper-toast.html

```diff
@@ -6,291 +6,138 @@
-    </paper-toast>
-    
-Positioning toast:
-
-A standard toast appears near the lower left of the screen.  You can change the
-position by overriding bottom and left positions.
-
-    paper-toast {
-      bottom: 40px;
-      left: 10px;
+<link rel="import" href="../polymer/polymer.html">
+
+<dom-module id="paper-toast">
+  <style>
+    :host {
+      display: inline-block;
+      background: #323232;
+      color: #f1f1f1;
+      min-height: 48px;
+      min-width: 288px;
+      padding: 16px 24px 12px;
+      position: fixed;
+      box-sizing: border-box;
+      box-shadow: 0 2px 5px 0 rgba(0, 0, 0, 0.26);
+      border-radius: 2px;
+      bottom: 12px;
+      left: 12px;
+      font-size: 14px;
+      cursor: default;
+      transition-duration: 300ms;
     }
-    
-To position the toast to the right:
 
-    paper-toast {
-      right: 10px;
-      left: auto;
-    } 
-  
-To make it fit at the bottom of the screen:
+    :host(.capsule) {
+      border-radius: 24px;
+    }
 
-    paper-toast {
+    :host(.fit-bottom) {
       bottom: 0;
       left: 0;
       width: 100%;
+      min-width: 0;
+      border-radius: 0;
     }
-  
-When the screen size is smaller than the `responsiveWidth` (default to 480px),
-the toast will automatically fits at the bottom of the screen.
-
-@group Paper Elements
-@element paper-toast
-@homepage github.io
--->
-<!--
-Fired when the `paper-toast`'s `opened` property changes.
-
-@event core-overlay-open
-@param {boolean} detail the opened state
--->
-<!--
-Fired when the `paper-toast` has completely opened.
-
-@event core-overlay-open-completed
--->
-<!--
-Fired when the `paper-toast` has completely closed.
-
-@event core-overlay-close-completed
--->
-<link rel="import" href="../core-overlay/core-overlay.html">
-<link rel="import" href="../core-transition/core-transition-css.html">
-<link rel="import" href="../core-media-query/core-media-query.html">
-
-<polymer-element name="paper-toast" attributes="text duration opened responsiveWidth swipeDisabled autoCloseDisabled" role="status">
-
-<template>
-
-  <link rel="stylesheet" href="paper-toast.css" >
-
-  <core-overlay id="overlay" autoFocusDisabled autoCloseDisabled="{{autoCloseDisabled}}" opened="{{opened}}" target="{{}}" transition="core-transition-bottom"></core-overlay>
-
-  <div class="toast-container" horizontal layout>
+    :host(.paper-toast-closed) {
+      visibility: hidden;
+      transition: visibility 300ms, transform 300ms;
+      transform: translateY(100px);
+    }
+    :host(.paper-toast-openend){
+      transition: visibility, transform, 300ms;
+      transform: translateY(-100px);
+    }
+  </style>
+  <template>
+    <span>{{text}}</span>
+    <content></content>
+  </template>
+</dom-module>
+<script>
+(function() {
 
-    <div class="toast-text" flex>{{text}}</div>
-    
-    <div class="toast-text toast-action" on-tap="{{dismiss}}">
-      <content></content>
-    </div>
+  var currentToast;
```

Comment:
Channeling @sjmiles for a second, let's keep this reference as a static property of the `PaperToast` constructor if possible.